docs/services/ml/upload-training-script.md

```diff
@@ -0,0 +1,216 @@
+---
+title: "Train a Model with a Custom Python Training Script"
+linkTitle: "Use a Custom Training Script"
+weight: 90
+type: "docs"
+tags: ["data management", "ml", "model training"]
+description: "How to write and upload a custom machine learning model training script to the Viam registry and use it to submit training jobs."
+no_service: true
+# SME: Tahiya S.
+---
+
+You can train your classification and object detection models on the [Viam app](https://app.viam.com), where the trained model is output as a TFLite model.
+See [Train a Model](/services/ml/train-model/) for more information.
+However, you can now also upload a custom training script to the [Viam registry](https://app.viam.com/registry/) to train your ML model with an output type of PyTorch, Tensorflow, TFLite, ONNX, or any other model artifacts.
+A flexible training interface like this allows you to upload any machine learning training code for your use-case.
```

Comment:
I'm not sure if the wording in the note would add more confusing because the process for that method is really different and you're not touching any code
```suggestion
Upload a custom training script to the [Viam registry](https://app.viam.com/registry/) to train your ML model with an output type of PyTorch, Tensorflow, TFLite, ONNX, or any other model artifacts.
This method allows you to upload any machine learning training code for your use-case.

{{< alert title="Note" color="note" >}}
For a lower-code approach, you can train your classification and object detection models on the [Viam app **DATA** page](https://app.viam.com), where the trained model is output as a TFLite model.
See [Train a Model](/services/ml/train-model/) for more information.
{{< /alert >}}
```
@npentrel Do you think this could work? Maybe with changing the phrase "lower code approach"

Reply:
I don't like the focus on the output format. I don't think that's what this is about. This is a way to let you run any Machine Learning training. So you could give it sensor data and create a model on that in theory I think.

Also I think this overuses possessive pronouns, I think I commented that elsewhere. And where  possible "See ... for more information" shouldn't exist in favour of the link being on the relevant information.

```suggestion
You can create your own custom Python training script that trains ML models to your specifications using the Machine Learning framework of your choice (PyTorch, Tensorflow, TFLite, ONNX, or any other framework).
Once added to the [Viam registry](https://app.viam.com/registry/), you can use the training script to build models for your datasets.

{{< alert title="Note" color="note" >}}
For a lower-code approach, you can [train TFLite classification and object detection models]((/services/ml/train-model/)) on the [Viam app **DATA** page](https://app.viam.com).
{{< /alert >}}
```

Reply:
@tahiyasalam maybe you can confirm that I am indeed understanding this correctly?

Reply:
I'm not sure I'm following this thread completely, but you can train any type of model, not just classification or object detection, and the model can be any outputted format (e.g. PyTorch, ONNX, TensorFlow).

Reply:
ahh, what is --model-type cli flag used to indicate or set then?

Reply:
I think that means my version of the copy works